erts/doc/src/erl_ext_dist.xml

```diff
@@ -1421,6 +1421,31 @@
       </note>
     </section>
 
+    <section>
+      <marker id="LOCAL_EXT"/>
+      <title>LOCAL_EXT</title>
+      <table align="left">
+	<row>
+	  <cell align="center">1</cell>
+	</row>
+	<row>
+	  <cell align="center"><c>121</c></cell>
+	</row>
+      <tcaption>LOCAL_EXT</tcaption></table>
+      <p>
+        Marks that this is encoded on a local term format intended to only be
+        decoded by the one that encoded it. The bytes following from here on
+        may contain any unspecified type of encoding of terms. For more
+        information see the
+        <seeerl marker="erts:erlang#term_to_binary_local"><c>local</c></seeerl>
+        option of 
+        <seemfa marker="erts:erlang#term_to_binary/2"><c>term_to_binary/2</c></seemfa>.
+      </p>
+      <p>
+        <c>LOCAL_EXT</c> was introduced in OTP @OTP-18477@.
+      </p>
```

Comment:
Would it make sense to explicitly clarify that after a restart the term won't be decodable? Or to otherwise specify what it meant by the same system that encoded it

Reply:
It is used that way by `term_to_binary()`, but it is not necessarily only used by the Erlang runtime system or any other OTP functionality which is why I was a bit fuzzy. It can perhaps be better explained, though.